armotypes/vulnerabilitytypes.go

```diff
@@ -9,11 +21,175 @@ type VulnerabilityJobParams struct {
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	RiskFactors      []RiskFactor               `json:"riskFactors"`
+	Labels           []string                   `json:"labels"`
+}
+
+type SeverityDetails struct {
+	Total   int             `json:"total"`
+	TopCves []TopCveDetails `json:"topCves"`
+}
+
+type TopCveDetails struct {
+	Name string `json:"name"`
+}
+
+type VulnerabilitiesImage struct {
+	ImageTag         string                     `json:"imageTag"`
+	ImageHash        string                     `json:"imageHash"`
+	ContainerName    string                     `json:"containerName"`
+	Registry         string                     `json:"registry"`
+	WorkloadsIds     []string                   `json:"wlids"`
+	NameSpace        string                     `json:"namespace"`
+	LastScan         time.Time                  `json:"lastScan"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+}
+
+// todo : check + rename
+type VulnerabilitiesComponent struct {
+	ImageTag         string                     `json:"imageTag"`
+	ImageHash        string                     `json:"imageHash"`
+	ContainerName    string                     `json:"containerName"`
+	Registry         string                     `json:"registry"`
+	WorkloadsIds     []string                   `json:"wlids"`
+	NameSpace        string                     `json:"namespace"`
+	LastScan         time.Time                  `json:"lastScan"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	Paths            []string                   `json:"paths"`
+}
+
 type Vulnerability struct {
-	Name          string   `json:"name"`
-	Severity      string   `json:"severity"`
-	SeverityScore int      `json:"severityScore"`
-	IsRCE         bool     `json:"isRCE"`
-	Links         []string `json:"links"`
-	Description   string   `json:"description"`
+	Id             string              `json:"id"`
+	Name           string              `json:"name"`
+	SeverityScore  int                 `json:"severityScore"`
+	IsRCE          bool                `json:"isRCE"`
+	Links          []string            `json:"links"`
+	Description    string              `json:"description"`
+	Exploitable    string              `json:"exploitable"`
+	ComponentsInfo []CveComponentsInfo `json:"componentsInfo"`
+	IsFixable      bool                `json:"isFixable"`
+	CvssInfo       []CvssInfo          `json:"cvssInfo"`
+	EpssInfo       EpssInfo            `json:"epssInfo"`
+	CisaKevInfo    CisaKevInfo         `json:"cisaKevInfo"`
```

Comment:
maybe inline?

Reply:
inline means the fields are under the object, here we want it as inner feild